### PR TITLE
Add highway=raceway to roads layer

### PR DESCRIPTION
**Pull request by rmarianski**
Connects to https://github.com/mapzen/vector-datasource/issues/664

### CODE REVIEW

File: test/664-raceway.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# https://www.openstreetmap.org/way/28825404
+assert_has_feature(
+    16, 10476, 25242, 'roads',
+    { 'id': 28825404, 'kind': 'minor_road', 'highway': 'raceway' })
+
+# Thunderoad Speedway Go-carts https://www.openstreetmap.org/way/59440900
```
Comment (nvkelso) [inline test/664-raceway.py]: Put the web link on it's own line.

**Comment by nvkelso** [comment]
Needs a custom `sort_key` value.

File: test/664-raceway.py (hunk 1 above)
Comment (rmarianski) [inline test/664-raceway.py]: Updated in 3bd8354.

**Comment by rmarianski** [comment]
> Needs a custom sort_key value.

Added in 038d56d.

File: test/664-raceway.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# https://www.openstreetmap.org/way/28825404
+assert_has_feature(
+    16, 10476, 25242, 'roads',
+    { 'id': 28825404, 'kind': 'minor_road', 'highway': 'raceway' })
```
Comment (nvkelso) [inline test/664-raceway.py]: Should we assert the `sort_key` value in the tests?

**Comment by nvkelso** [comment]
Update the tests, then 👍.

File: test/664-raceway.py (hunk 2 above)
Comment (rmarianski) [inline test/664-raceway.py]: > Should we assert the sort_key value in the tests?

We could for now, knowing that it makes the tests more brittle.

IIUC, what we really care about here with sort keys is the relative positioning between certain kinds of features. In the future, I'd rather that we created generative tests here instead and had the relative feature positions be our properties/invariants.

Comment (nvkelso) [inline test/664-raceway.py]: I'm fine with brittle for now. File a new issue for the generative tests for the future?

Comment (rmarianski) [inline test/664-raceway.py]: New issue: https://github.com/mapzen/vector-datasource/issues/805

**Comment by rmarianski** [comment]
Tests updated in 3bbfc62.

**Pull request merged**